console_backend/src/tabs/settings_tab.rs

```diff
@@ -599,18 +602,17 @@ impl SettingsTab {
         let original = self
             .settings
             .lock()
-            .get(group, name)?
-            .value
-            .as_ref()
-            .ok_or_else(|| anyhow!("{group}.{name} was none"))?
-            .clone();
+            .get(group, name)
+            .map(|x| x.value.clone())
+            .unwrap_or_default();
```

Comment:
this was the second issue, `ntrip.enable` field was not found in old values and would terminate here.
this now assumes the groups are off by default so we won't unset it